bgpd/bgp_vty.c

```diff
@@ -1962,7 +1962,7 @@ DEFUN (bgp_graceful_restart_stalepath_time,
 
 DEFUN (bgp_graceful_restart_restart_time,
        bgp_graceful_restart_restart_time_cmd,
-       "bgp graceful-restart restart-time (1-3600)",
+       "bgp graceful-restart restart-time (1-4095)",
```

Comment:
The BGP YANG model from IETF is using a range of `0..4096` for this configuration option. Would it be possible to use that range here too? If `0` conveys special meaning, then maybe `1..4096` would be fine as well.

Reference: https://github.com/YangModels/yang/blob/master/experimental/ietf-extracted-YANG-modules/ietf-bgp-common%402018-05-09.yang#L232-L242

Reply:
just to be sure: 4096 can not be encoded on on 12 bit, since it takes exactly 13 bits.
and as per rfc4724, restart time is encode on 12 bits.
am I right to say there is something wrong with either YangIETF or RFC ?

Reply:
@louberger @riw777 any thoughts about this problem?

Reply:
90% sure this is a bug in the YANG model :disappointed: